build.gradle

```diff
@@ -1,23 +1,26 @@
 plugins {
-  id 'com.diffplug.spotless' version '5.12.1'
+  id 'com.diffplug.spotless' version '6.10.0'
```

Comment:
[6.11.0](https://plugins.gradle.org/plugin/com.diffplug.gradle.spotless) is available if you like

Reply:
Yep -- let's go with the latest here 👍